### ISSUE TITLE
Send multiple data

### ISSUE THREAD

**Issue by apaini** [jamontes79/xamarin-forms-firebase-sample#1]
How can I send multiple data with model. it's like 
User User{
id = "userid"
name = "username"
}

**Comment by jamontes79** [comment]
Hi,

The idea is convert the object into json and send to Fibrease.

This in an example from another project, I hope it can help you:

`
 private void InsertPrivateList(ListItem listItem)
        {
            var userId = authService.GetUserId();
            databaseReference = database.GetReference(userId + "/privateLists/");

            var newItem = databaseReference.Push();
            listItem.Id = newItem.Key;
            string json = JsonConvert.SerializeObject(listItem);
            newItem.SetValue(json);
        }

`

Regards

**Comment by jamontes79** [comment]
Hi @apaini did my last comment help you?

Thanks
Regards

**Comment by apaini** [comment]
@jamontes79 yes, but not really. i think it's tricky rather than solution. i'm still figure it out how to use HashMap in xamarin. check this out https://forums.xamarin.com/discussion/84732/firebase-database-xamarin-implementation
